fe2-android/app/src/main/java/com/github/dedis/popstellar/model/objects/Lao.java

```diff
@@ -48,7 +48,8 @@ public Lao(String id) {
     } else if (id.isEmpty()) {
       throw new IllegalArgumentException(" The id of the Lao is empty");
     }
-    this.channel = "/root/" + id;
+
+    this.channel = Channel.ROOT.sub(id);
```

Comment:
this is not really more understandable than before...

Reply:
It's true that it is not that much more readable, but this can still be improved.

I mostly focused on constancy and robustness when designing the creation API because for me, the way channels were created was already decent except for the fact that is used "magic constants".

Reply:
That's how I read it. I agree it was already much better, just not much more readable (yet) 👍